sdk/test-utils/recorder/README.md

```diff
@@ -1,14 +1,26 @@
 ## Azure @azure-tools/test-recorder-new library for JavaScript
 
-This is an experimental tool to record and playback the tests in the JS repo by leveraging the unified out-of-process test proxy server. This library is still under construction.
-Feature work is being tracked at [#15829](https://github.com/Azure/azure-sdk-for-js/issues/15829)
+**Note: This project is a test utility that assits with testing the packages maintained at the Azure SDK for JavaScript repository. This is not intended for the public utilization.**
+
+This tool helps to record and playback the tests in the JS repo by leveraging the unified out-of-process test proxy server.
 
 ## Resources
 
+- [Migration guide to recorder v2 from v1](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/MIGRATION.md)
 - [Azure SDK Tools Test Proxy](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy)
 - [Using Test Proxy with docker container](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy/docker#build-and-run)
 
-## Running the test-proxy tool
+## Running the tests along with the test-proxy tool
+
+### With the `dev-tool` commands
+
+- The following commands run the tests with the default configs, and concurrently runs the proxy tool in record/playback modes if it is not already active. Additionally, more options can be passeed to override the default configs.
+  - `dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'`
```

Comment:
Since this will now be a public package, I would move this section as the last option and make a note stating that dev-tool should only be used inside the repo and there are no plans to make it publicly available.

Reply:
It's not a public package yet.
So, I think this is fine for now since we have the 
> "**Note: This project is a test utility that assits with testing the packages maintained at the Azure SDK for JavaScript repository. This is not intended for the public utilization.**" 

at the beginning of the readme,